changelog.d/20230527_174319_codewithemad_kubeconfig.md

```diff
@@ -0,0 +1 @@
+[Bugfix] Prioritize loading of incluster_config over kube_config. In certain scenarios, Tutor may operate within a pod that has access to a cluster through role binding and a service account. In these instances, the ./kube/config file may not be present, but kubectl commands can still execute without any problems. (by @CodeWithEmad)
```

Comment:
nit: add a hyphen "-" at the beginning of the line because the changelog entries are collected into an item list.

Reply:
> The functions are not used anywhere else, and they are quite simple, so I figure we can just remove them.

I agree with you. I made those changes to make things clearer. It's no problem to remove the functions.

> if we detect early that there is no kube config file or cluster env vars, then we should raise a TutorException that will be caught by the Tutor main process

Totally Agree.

Reply:
Updated,
is it necessary to update documents on k8s?

Reply:
No, we are good to go :) Thanks!